### PR TITLE
fix #69 by setting preferredLineEnding

### LINKED ISSUE #69: Opening files automatically registers a change
When you open any file with editorconfig enabled, Atom registers a change to the file instantly. As well as being quite annoying, it makes it difficult to track what you have and haven't made changes to, When asked if I want to save my changes, I'm not always sure that I have changed anything, or if it's just editorconfig.

### PR DESCRIPTION
**Pull request by florianb**
Instead of the previous char-replace in the buffer.

### CODE REVIEW

**Comment by Arcanemagus** [comment]
Personally I loved this "bug" as it nicely exposed repos that weren't set up properly yet, but this is how an editor config package is _supposed_ to work so :+1: :stuck_out_tongue:

**Comment by florianb** [comment]
That commet should have gone into issue #69 - since personally would also prefer to be warned about that issue i could imagine an option which warns about inconsistencies.

**Comment by Arcanemagus** [comment]
Unfortunately as the `editorconfig` issue linked there states, transforming a file with improper line endings isn't what a "proper" `editorconfig` implementation should do :disappointed:.

**Comment by florianb** [comment]
Right - that's why probably an enhancement with proper **notifications** should be filed.

File: spec/iss69-spec.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	describe('when opening any file', () => {
+		const filePath = path.join(projectRoot, 'iss69.txt');
+		let textEditor = null;
+
+		atom.workspace.open(filePath)
+			.then(newTextEditor => {
+				textEditor = newTextEditor;
+			});
+
+		it('shouldn\'t being changed without any action', () => {
```
Comment (glen-84) [inline spec/iss69-spec.js]: `shouldn\'t being changed` -> `shouldn\'t be changed`

**Comment by florianb** [comment]
Interesting - switching during a squash closes the PR?

**Comment by florianb** [comment]
Thanks everybody for support! 👏

**Pull request merged**